### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by fia5000** [kacole2/JumpSquares#2]
I've tried many times to get this to work but am plagued with errors like undefined method `sortorder' for nil:NilClass app/controllers/jumpsquares_controller.rb:9:in`index'

I was trying to build a docker image out of this but the app itself does not run. I am not sure how the appliance version works off this same code base. 

Albeit, this is more than 2 years old and seemingly abandoned, on wheezy that is no longer supported by docker.

if you ever find the time and want to revive this project, let me know.

I made some changes to the front-end on my running appliance version http://i.imgur.com/sefHbpM.png

**Comment by kacole2** [comment]
@kilokill yes the project is basically abandoned because I don't have time for it and I never knew the amount of traction it had. Perhaps it can pick it back up. 

I think a Dockerized version could be in order

At this time, I really only monitor and mange the hosted version at Jumpsquares.net

I really love what you did with the front end UI! I would encourage you to do a Pull Request with those changes

So let me pick this is up next week and see what I can do. I can imagine so much will be broken after not having a single package updated in two years.  

I know Solr was a requirement for database searching so make sure that is installed.

**Comment by fia5000** [comment]
@kacole2 Thanks for the reply. I tried to, well, re-trace your steps in the appliance version by looking at postgres and bash history but just couldn't get it to work. I am not sure why there are so many method not defined errors coming up, when the same code works in the virtual machine version.

I suspect it may have to do with version of ruby being used by me on the docker rails image.

I think this project has a lot of potential - Nothing like it exists and if revived, I can see a lot of corporate admins making this a mainstay in their orgs.

The hosted version is good, I've thought of using it - the barrier is the obvious elephant in the room when pitching this to bosses, who question potentially sensitive data being in the "cloud".  Of course, you don't have that problem with homelabs etc.

**Comment by kral2** [comment]
Hi,

Is there any chance to get a revival on this hidden gem? :-)
@fia5000 your modifications to the frontend are nice. Are you still maintaining these changes or willing to share it?

**Comment by kacole2** [comment]
@kral2 sorry. No time right now. Everything is open source so it should all be available in this repo. 

The online SaaS version at jumpsquares.net is still alive and kicking though.

**Comment by kacole2** [comment]
There's also no pull requests being pushed for fixes. I'm willing to add someone as a new maintainer. I just don't have the cycles.

**Comment by kral2** [comment]
I am willing to invest some time and get hands on with ror, so this could be a useful learning project.

Project forked, I will move at my own pace and keep you updated if I manage to get results.

Your code seems well commented, but any context you can give me to jumpstart is obviously welcome.
